compute_endpoint/globus_compute_endpoint/endpoint/interchange.py

```diff
@@ -435,6 +432,13 @@ def process_pending_results():
                         message = try_convert_to_messagepack(packed_result)
 
                     except Exception as exc:
+                        if not task_id:
```

Comment:
Swallowing exceptions is often questionable!  Namely when displaying a terse error message.